src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -227,8 +227,9 @@ def load_arguments(self, _):
             c.argument('settings', nargs='+', help="space-separated app settings in a format of `<name>=<value>`")
             c.argument('setting_names', nargs='+', help="space-separated app setting names")
         with self.argument_context(scope + ' config ssl import') as c:
+            c.argument('certificate', help='The name or resource ID of the certificate to be imported')
+            c.argument('key_vault_certificate_name', help='The name of the certificate in Key Vault', deprecate_info=c.deprecate(expiration='2.18.0', target='--key-vault-certificate-name', redirect='--certificate-name'))
```

Comment:
check what the most recent build version is - incase the 2.18.0 needs to be updated.